project/src/main/chat/ui/touch-translator.gd

```diff
@@ -7,6 +7,9 @@ export (NodePath) var narration_frame_path: NodePath
 ## index of the current touch event, or -1 if there is none
 var _touch_index := -1
 
+## 'true' if the player is making a dialog choice
+var _showing_choices := false
```

Comment:
This field is never used.

Reply:
Unfortunately, it's used on line 76 which was already merged (and which does not compile without this change, oops.)